aion-etl/src/main/java/aion/dashboard/service/DBService.java

```diff
@@ -32,6 +53,99 @@ public BigDecimal getCirculatingSupply(Instant timestamp) throws SQLException {
         }
         return circulatingSupply.floorEntry(timestamp).getValue();
     }
+/*
+* This method will calculate the Total circulation Supply by the sum of all the blocks reward plus the networkBalanceAlloc.
+* Total circulation Supply will be represented by the current height-4320 blocks  (aprox 1/2 day)
+* @author Jesus Redon
+* */
+    public void generateCirculatingSupply() throws SQLException {
+
+        long hight=blockService.getLastBlockNumber()-4320l;
```

Comment:
Just curious, any reason for picking this value (4320)?

Reply:
Just to get the CS without the last 6 hours, no particular reason, would you like to make it shorter ?

Reply:
Nope all good, just wanted to understand the reasoning